### PR TITLE
Fixed DTK_LIBRARIES patch issue (again)

### PR DESCRIPTION
**Pull request by fcollot**
Last week, PR https://github.com/Inria-Asclepios/medinria-superproject/pull/24 introduced a faulty patch that prevented MUSIC from compiling. PR https://github.com/Inria-Asclepios/medinria-superproject/pull/26 adds a second patch that fixes the original issue, but there is a bug in the superproject that prevents multiple patches from modifying the same file in one build. As a result, the second patch is not applied on Jenkins and so the original patch still prevents compilation.
This third PR merges the two patches together to make sure both changes are applied. I'm not in favour of merging patches like this but I don't have time to find a better solution.

### CODE REVIEW

**Comment by fcollot** [comment]
(I tried to be clear in the header because 6 months from now this will be hard to follow)

File: patches/dtkLibraries.patch [hunk 1]
```diff
@@ -1,36 +1,34 @@
...
-Subject: [PATCH] dtkLibraries
+commit c214fd6e5e11fc011f02ece457e9a82b260e046d
+Author: Florent Collot <florent.collot@ihu-liryc.fr>
+Date:   Fri Sep 28 15:04:27 2018 +0200
 
----
- cmake/FindDTK.cmake.in   | 2 +-
- cmake/dtkConfig.cmake.in | 2 ++
- 2 files changed, 3 insertions(+), 1 deletion(-)
+    foo
```
Comment (mathildemerle) [inline patches/dtkLibraries.patch]: "foo" here, is it normal?

Comment (fcollot) [reply patches/dtkLibraries.patch]: The method I use to create the patch needs to have the changes commited, and since it is just a dummy commit to obtain the patch, I named it foo. The patch generator puts the commit name in the patch but it has no effect.

**Comment by fcollot** [comment]
oh wait this doesn't remove the second patch. One second.

**Comment by fcollot** [comment]
updated

**Review by mjuhoor — APPROVED** [review]

File: patches/dtkLibraries.patch [hunk 2]
```diff
@@ -1,36 +1,34 @@
...
++++ b/cmake/dtkUse.cmake.in
+@@ -31,4 +31,10 @@ if(NOT DTK_USE_FILE_INCLUDED)
+    include(dtkBuild)
+    include(dtkReporting)
  
- set(DTK_BUILD_VR "@DTK_BUILD_VR@")
- set(DTK_BUILD_WRAPPERS "@DTK_BUILD_WRAPPERS@")
++   set(DTK_LIBRARIES dtkMath dtkCore dtkGui dtkLog)
 +
-+set(DTK_LIBRARIES dtkMath dtkCore dtkGui dtkComposer dtkLog)
```
Comment (mathildemerle) [inline patches/dtkLibraries.patch:33]: dtkComposer is still here, is it normal ?

Comment (mathildemerle) [reply patches/dtkLibraries.patch:33]: oups it's a " - +" ok

**Review by mathildemerle — APPROVED** [review]

**Review by juliencastelneau — APPROVED** [review]

**Comment by mathildemerle** [comment]
Oups, in dtk.cmake, the code is still:

```
ep_GeneratePatchCommand(DTK DTK_PATCH_COMMAND
    dtkLibraries.patch
    dtkLibraries2.patch)
```
